sandbox/forms/forms-sandbox.js

```diff
@@ -0,0 +1,189 @@
+import { css, html, LitElement } from 'lit';
+import { html as staticHtml, unsafeStatic } from 'lit/static-html.js';
+import '../../src/components/cc-select/cc-select.js';
+import '../../src/components/cc-smart-container/cc-smart-container.js';
+
+const DEMOS = {
+  'with-cc-components': 'With clever components',
+  'with-native-inputs': 'With native inputs',
+  reset: 'With reset values',
+  'with-array-type': 'With array types',
+  'with-custom-error-message': 'With custom error messages',
+  'with-custom-validation': 'With custom validations',
+  'with-coupled-inputs': 'With coupled inputs',
+  'with-fieldset': 'With fieldset',
+  'dynamic-form': 'With dynamic form',
+  'with-smart-component': 'With smart component',
+};
```

Comment:
**👏 praise:** Thank you very much for the demos, they are really helpful. :fire: